sdk/src/types/block/mana/structure.rs

```diff
@@ -73,13 +69,11 @@ impl ManaStructure {
             // Unwrap: Safe because the index is at most the length
             let decay_factor = self.decay_factor_at(epochs_to_decay - 1).unwrap();
 
-            // apply the decay using fixed-point arithmetics.
-            (mana_hi, mana_lo) =
-                multiplication_and_shift(mana_hi, mana_lo, decay_factor, self.decay_factors_exponent());
+            // apply the decay using fixed-point arithmetics
```

Comment:
It's not really using fixed point anymore?

Reply:
I'm not entirely sure it was ever fixed point math 🤷‍♀️